Scripts/Knight.gd

```diff
@@ -15,8 +17,6 @@ var defense := 1
 var attack := 1
```

Comment:
attack is ambiguous here since it is also a bern. Maybe append stat to the end of the stats.